recipes/resiprocate/all/conanfile.py

```diff
@@ -29,13 +29,22 @@ class ResiprocateConan(ConanFile):
     def _source_subfolder(self):
         return "source_subfolder"
 
-    def requirements(self):
+    def config_options(self):
+        if self.settings.os == 'Windows':
+            del self.options.fPIC
+
+    def configure(self):
         if self.settings.os in ("Windows", "Macos"):
-            raise ConanInvalidConfiguration("reSIProcate is not support on {}.".format(self.settings.os))
+            # FIXME: Visual Studio project & Mac support seems available in resiprocate
+            raise ConanInvalidConfiguration("reSIProcate recipe does not currently support {}.".format(self.settings.os))
+        if self.options.shared and self.options.fPIC:
+            raise ConanInvalidConfiguration("fPIC option should be False when shared option is True")
```

Comment:
this is how we usually do it

https://github.com/conan-io/conan-center-index/blob/689e2663fe41689259f61a5dbd3fe245e24c03b9/recipes/wslay/all/conanfile.py#L37-L38